hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc

```diff
@@ -178,6 +178,13 @@ Additionally, if the vertex sourcing them is a data source or data sink, then th
 |===
 |Names|Extra tags
 
+|**executionStartTime**: Start time of the current execution of a job. +
+**executionCompletionTime**: Start time of the current execution of a job. +
+
+Both contain epoch time in milliseconds.
+|** * only ones without `vertex` tag** +
```

Comment:
what does this mean?

Reply:
See the first paragraph of the "Jet Job metrics" section, stating that almost all job metrics have the 'vertex' tag set (ie. are per vertex). So I wanted to point out that these two metrics are the exceptions.

Reply:
Pushed a more straightforward version.